### ISSUE TITLE
ERROR missing field "node" when creating decision

### ISSUE THREAD

**Issue by ylaizet** [gorules/zen#126]
I got the following error  when I tried to create a decision from json rules files created on https://editor.gorules.io/ with python zen-engine==0.18.1 

```decision = engine.create_decision(content)
RuntimeError: Failed to serialize decision content

Caused by:
    missing field `nodes` at line 6 column 1
```
It's due to the order of the keys in the json rules created: if the key "nodes" is after the "edges" node, the error is raised. If I put "node" before "edges", it works.

On https://editor.gorules.io, the simulator works without problem with the same json rules files.

**Comment by ylaizet** [comment]
Sorry, my bad it works whatever is the order. I mixed filenames

**Comment by xplore42** [comment]
@ylaizet Hi, I am also facing same issue when running it from nodeJs. could you please share how it solved for you.
